sound/soc/sof/intel/hda-dsp.c

```diff
@@ -375,6 +395,18 @@ static int hda_resume(struct snd_sof_dev *sdev)
 	snd_hdac_ext_bus_ppcap_enable(bus, true);
 	snd_hdac_ext_bus_ppcap_int_enable(bus, true);
 
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_HDA_AUDIO_CODEC)
+	if (resume_runtime) {
```

Comment:
I assume reading STATESTS here wouldn't work? It has to be done earlier?

Reply:
STATESTS would be cleared if chip is initialized, so read if before chip initialization.